login-workflow/example/src/screens/ExampleRegistrationSuccess.tsx

```diff
@@ -9,91 +9,99 @@ type CustomRegistrationProps = {
     registrationData: RegistrationData;
 };
 
-export const CustomRegistrationSuccessScreen: React.FC<CustomRegistrationProps> = (props) => (
-    <>
-        <CardHeader
-            title={
-                <Typography variant={'h6'} style={{ fontWeight: 600 }}>
-                    Custom Registration Success Screen
-                </Typography>
-            }
-            style={{ padding: '16px 24px' }}
-        />
-        <CardContent
-            style={{
-                flex: '1 1 0px',
-                overflow: 'auto',
-                display: 'flex',
-                flexDirection: 'column',
-                justifyContent: 'center',
-                alignItems: 'center',
-                height: '100%',
-                padding: '16px 24px',
-            }}
-        >
-            <EmptyState
-                icon={<Person color={'primary'} style={{ fontSize: 100 }} />}
-                title={`Congratulations ${props.registrationData?.accountDetails?.firstName || ''}${' '}
-                ${props.registrationData?.accountDetails?.lastName || ''}!`}
-                description={'You made it to the custom registration success screen!'}
+export const CustomRegistrationSuccessScreen: React.FC<CustomRegistrationProps> = (props) => {
+    const matchesXS = useMediaQuery('(max-width:600px)');
+
+    return (
+        <>
+            <CardHeader
+                title={
+                    <Typography variant={'h6'} style={{ fontWeight: 600 }}>
+                        Custom Registration Success Screen
+                    </Typography>
+                }
+                style={{ padding: matchesXS ? '16px 24px 0 24px' : '32px 24px 0 24px' }}
```

Comment:
use `theme.spacing()` for horizontal paddings, `1.5rem` for vertical ones

Reply:
This might apply to https://github.com/pxblue/react-workflows/pull/47/files?file-filters%5B%5D=.tsx#diff-7e37bb22f50be7f24607abeed7d52403f0ae5f32dff9ca0dd4055a68eb209b5aR6-R8